03.asynchronous/callback.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books(id INTEGER PRIMARY KEY, title TEXT NOT NULL UNIQUE)",
+  () => {
+    db.run("INSERT INTO books (title) VALUES (?)", ["book1"], () => {
+      db.run("SELECT * FROM books", function () {
+        console.log(this.lastID);
+
+        db.get("SELECT * FROM books where title=?", ["book1"], (_, row) => {
```

Comment:
スタイルが整っていないです。また、SQL のキーワードは大文字で書きましょう。

```suggestion
        db.get("SELECT * FROM books WHERE title = ?", ["book1"], (_, row) => {
```